storage/indexer/src/db_indexer.rs

```diff
@@ -182,7 +202,18 @@ impl DBIndexer {
                         }
                     });
                 }
+
+                if self.config.enable_statekeys {
+                    writeset.iter().for_each(|(state_key, write_op)| {
+                        if !write_op.is_deletion() {
```

Comment:
no I mean modifications don't need to be writen as well. out of "creation", "modification", and "deletion", only creations need to be written, right? -- you'll save a lot of writes